versions/1/changelog.yml

```diff
@@ -52,3 +52,6 @@
   - description: Add fields for matching datasets dynamically
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/102
+  - description: Replace ioutil for Go1.16 update
+    type: breaking-change
```

Comment:
I'm not sure this is a breaking change. No public APIs have changed in this PR. I would just make this `type: enhancement` for now.